test/integration/buyers/service_contracts_controller_test.rb

```diff
@@ -92,6 +92,19 @@ class ProviderAdminTest < self
       page = Nokogiri::HTML4::Document.parse(response.body)
       assert page.xpath("//select[@id='service_contract_plan_id']/option").map(&:text).exclude?('Please select')
     end
+
+    test 'no n+1 queries on index' do
+      populate = ->(n) do
+        n.times do
+          buyer = FactoryBot.create(:buyer_account, provider_account: provider)
+          service_contract = FactoryBot.create(:simple_service_contract, plan: service_plan, user_account: buyer)
```

Comment:
Create list could be simpler I guess but just a nitpick.

I'm just worried that without a warmup things have failed for me in the past with oracle. So please try how this works in oracle also.